### DISCUSSION TITLE
Installation guide

### DISCUSSION THREAD

**Discussion by gabonator** [gabonator/LA104#49] [General]
...share your experiences and problems here...

**Comment by JonCyberman** [comment]
Does anyone have the link to buy the exact module Gabriel uses? I have several different modules with similar references, they all detect the signals but it always says "unknown protocol", I have tested several transmission devices , Thank you

**Reply by gabonator** [reply]
I am using this module (but any CC1101 based module should work): https://www.ebay.com/itm/315-433-868-915Mhz-CC1101-Wireless-Module-SMA-Antenna-Transceiver/233768387701 
This device is tuned to 433 MHz, but if you place the transmitter next to the antenna, you will also receive 868 MHz traffic

**Comment by dhosenfeld** [comment]
Hello,
first of all, thanks for your work.
I've tried to install your system for the LA104 in the version of March 24.
The systems seems to be installed and after the restart the device asks for the shell.elf image.
I used dfuload to copy the shell.elf from the tool directory to the device in two steps:

1. I unmount disk2 and then 
2. I copied shell.elf to the device

The dfuload seems to work correct. But on the device I get a new file called A.ERR.
After restarting the device asks for shell.elf again.
That's the point were I am. Does anyone have a suggestion what I can do?

Thank you
Dirk

**Reply by gabonator** [reply]
dfuload is only for initial installation of system.hex file. After reboot just copy the shell.elf over the usb cable as if you were copying to regular usb drive (use your favourite file manager or windows explorer)

**Reply by dhosenfeld** [reply]
Hello gabonator,

Thanks, now it Wwrks.
Only one question: Is it right that I have to shut off the devise to close the analyser?

Dirk

**Reply by gabonator** [reply]
There should be EXIT item in file menu, but I cant remember exactly... But turning off the device is faster way :)

**Reply by dhosenfeld** [reply]
Thank you. :)
